src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -39,6 +39,9 @@
     <file src="GenFacades\GenFacades.exe" target="lib" />
     <file src="GenAPI\GenAPI.exe" target="lib" />
     <file src="ApiCompat\ApiCompat.exe" target="lib" />
+    <file src="Microsoft.DotNet.CodeAnalysis\Microsoft.DotNet.CodeAnalysis.dll" target="lib" />
```

Comment:
Are all the other necessary dependencies already included into the package directories?